opencood/data_utils/datasets/__init__.py

```diff
@@ -1,11 +1,13 @@
 from opencood.data_utils.datasets.late_fusion_dataset import LateFusionDataset
 from opencood.data_utils.datasets.early_fusion_dataset import EarlyFusionDataset
 from opencood.data_utils.datasets.intermediate_fusion_dataset import IntermediateFusionDataset
+from opencood.data_utils.datasets.intermediate_fusion_dataset_v2 import IntermediateFusionDatasetV2
```

Comment:
So you did not merge IntermediateFusionDataset and V2 into one?

Reply:
yeah, i am thinking maybe keep both of them. How do you think